examples/coordinates/plot_galactocentric-frame.py

```diff
@@ -21,13 +21,13 @@
 Note that, for brevity, we may refer to the solar system barycenter as just "the
 sun" in the examples below.
 
--------------------
+---
```

Comment:
Huh... What does this mean? Does `---` mean something special?

Reply:
Hmm well it stopped the warnings about these headings but the output doesn't look good. Fixing...

Reply:
I just went ahead and removed them. Basically Sphinx thinks they are heading lines and complains about them.